package.json

```diff
@@ -1,11 +1,14 @@
 {
   "private": true,
   "scripts": {
-    "prepare": "lerna run prepare",
-    "rebuild:browser": "theia rebuild:browser"
+    "prepare": "yarn tslint && lerna run prepare",
+    "rebuild:browser": "theia rebuild:browser",
+    "tslint-fix": "tslint --fix --project theia-dashboard-extension",
+    "tslint": "tslint --project theia-dashboard-extension"
   },
+  "license": "EPL-2.0",
   "devDependencies": {
-    "typescript": "2.7.2",
```

Comment:
could you explain why we remove this dependency ? is it related to tslint config ?

Reply:
It is a line with code clean up. We have needed version in [theia-dashboard-extension/package.json](https://github.com/eclipse/che-theia-dashboard-extension/blob/master/theia-dashboard-extension/package.json). I just forgot about it after some experiments with versions.

Reply:
ok, you should make it a separate commit or PR, as you wish.

Reply:
done